### PR TITLE
Release 1.7.1

### PR DESCRIPTION
**Pull request by ssbarnea**

### CODE REVIEW

**Comment by shawnrusaw-wf** [comment]
I'm really looking forward to `--directory` option. Thx!

**Comment by insspb** [comment]
@ssbarnea 
Not sure about 1.7.2
As it seems life introduced new problems and fast releases currently not possible. Maybe we should concentrate on 2.0 or 1.8? Like in project plna. What you think? 
Catch me in slack to talk.

File: HISTORY.md [hunk 1]
```diff
@@ -2,7 +2,10 @@
 
 History is important, but our current roadmap can be found [here](https://github.com/cookiecutter/cookiecutter/projects)
 
-## 1.7.1 (Current master, in development)
+## 1.7.2 (Current master, in development)
+* ...
+
+## 1.7.1 ((2020-03-30)
```
Comment (insspb) [inline HISTORY.md:8]:
```suggestion
## 1.7.1 (2020-04-17)
```

**Comment by insspb** [comment]
@ssbarnea I will close it, just because I will make release manually this time. Do not waste you time on this :) Thanks

**Pull request closed without merging**